1.1/spec/20-Annex-E.adoc

```diff
@@ -35,9 +35,9 @@ The LOCN specification provides notes on the use of GeoSPARQL literals (see http
 | From Element | Mapping relation | To Element | Notes
 
 | `geo:Feature` | `rdfs:subClassOf` | `dcterms:Location` | LOCN states that `dcterms:Location` "represents any location, irrespective of size or other restriction", thus it is a superclass of `geo:Feature` 
-| `locn:Address` | `rdfs:subClassOf` | `geo:Feature` | Although LOCN indicates no spatial or geometry properties for `locn:Address`, `locn:Address` is clearly some specailized form of a Feature
-| `geo:Geometry` | `rdfs:subClassOf` | `locn:Geometry` | The LOCN class "defines the notion of "geometry" at the conceptual level, and it shall be encoded by using different formats", so GeoSPARQL's class is more specialised.
-| `geo:hasGeometry` | `rdfs:subPropertyOf` | `locn:geometry` | The LOCN property notes say "Depending on how a geometry is encoded, the range of this property may be one of..." so GeoSPARQL's proerty is learly more specialized.
+| `locn:Address` | `rdfs:subClassOf` | `geo:Feature` | Although LOCN indicates no spatial or geometry properties for `locn:Address`, `locn:Address` is clearly some specialized form of a Feature
+| `geo:Geometry` | `owl:equivalentClass` | `locn:Geometry` | The LOCN class "defines the notion of "geometry" at the conceptual level, and it shall be encoded by using different formats", so GeoSPARQL's class is equivalent.
+| `geo:hasGeometry` | `owl:equivalent` | `locn:geometry` | The LOCN property notes say "Depending on how a geometry is encoded, the range of this property may be one of..." so GeoSPARQL's property is clearly more specialized.
```

Comment:
`owl:equivalentProperty`

Reply:
Adopted the first - `geo:Geometry owl:equivalentClass locn:Geometry` but not the second `geo:hasGeometry owl:equivalent[Property] locn:geometry` as I've chosen to keep using `rdfs:subPropertyOf` since GeoSPARQL's version of this property is clearly more specialized!